src/user.ts

```diff
@@ -61,6 +61,13 @@ export function ChangePassword(this: LynxClient, passwordData: ChangePasswordDat
     });
 }
 
+export function ChangePasswordOther(this: LynxClient, newPassword: string) {
+    return this.requestJson<OKResponse>(`${Endpoints.User}/password`, {
+        method: 'PUT',
+        body: JSON.stringify(newPassword),
```

Comment:
This would just create the JSON `"<password>"`, it should be placed in an object with the password key to produce `{"password": "<password>"}`